src/plfa/part2/DeBruijn.lagda.md

```diff
@@ -271,6 +272,16 @@ is a context with two variables in scope, where the outer
 bound one has type `` `ℕ ⇒ `ℕ ``, and the inner bound one has
 type `` `ℕ ``.
 
+We define a helper function that computes the length of a context,
+which will be useful later in making sure an index is within context
+bounds:
```

Comment:
If it isn't going to be used until later, than perhaps we shouldn't define it until it is useful.

Reply:
Sure, moving it to just before the formalisation of `lookup`.